website/javascript/templates/Home.vue

```diff
@@ -19,7 +19,11 @@
                     Build a bot and get on the leaderboard.<br/>
                     Check out our documentation and interactive tutorials.
                 </div>
-                <div class="submit-bot">
+                <a
+                  v-if="has_bots"
```

Comment:
Does v-if="user.rank" above have us covered for this use case?

Reply:
I think during compilation, you can have a bot but not a rank.